tests/validation/tests/v3_api/common.py

```diff
@@ -1603,23 +1619,40 @@ def validate_catalog_app(proj_client, app, external_id, answer=None):
         "the version of the app is not correct"
     # check if associated workloads are active
     ns = app.targetNamespace
-    pramaters = external_id.split('&')
-    assert len(pramaters) > 1, \
-        "Incorrect list of paramaters from catalog external ID"
-    chart = pramaters[len(pramaters)-2].split("=")[1] + "-" + \
-            pramaters[len(pramaters)-1].split("=")[1]
+    parameters = external_id.split('&')
+    assert len(parameters) > 1, \
+        "Incorrect list of parameters from catalog external ID"
+    chart = parameters[len(parameters)-2].split("=")[1] + "-" + \
+            parameters[len(parameters)-1].split("=")[1]
+    app_name = parameters[len(parameters)-2].split("=")[1]
     workloads = proj_client.list_workload(namespaceId=ns).data
     for wl in workloads:
         assert wl.state == "active"
-        assert wl.workloadLabels.chart == chart, \
-            "the chart version is wrong"
+        chart_deployed = get_chart_info(wl.workloadLabels)
+        if app_name in chart_deployed and '-' in chart_deployed:
+            assert chart_deployed == chart, "the chart version is wrong"
 
     # Validate_app_answers
     assert len(answers.items() - app["answers"].items()) == 0, \
         "Answers are not same as the original catalog answers"
     return app
 
 
+def get_chart_info(workloadlabels):
+    """
+    This method finds either 'chart' tag or
+    'helm.sh/chart' tag from workload API
+    @param workloadlabels:
+    @return: chart value of workload e.g. 'app_name-version'
+    """
+    if "chart" in workloadlabels.keys():
+        return workloadlabels.chart
+    elif "helm.sh/chart" in workloadlabels.keys():
```

Comment:
When does this scenario happen?

Reply:
There are "helm.sh/chart" tag instead of 'chart' tag in some cases.